examples/int32_publisher/main/rclc_int32_publisher.c

```diff
@@ -38,8 +38,8 @@ void appMain(void * arg)
 	rmw_init_options_t* rmw_options = rcl_init_options_get_rmw_init_options(&init_options);
 
 	// Static Agent IP and port can be used instead of autodisvery.
-	// RCCHECK(rmw_uros_options_set_udp_address(CONFIG_MICRO_ROS_AGENT_IP, CONFIG_MICRO_ROS_AGENT_PORT, rmw_options));
-	RCCHECK(rmw_uros_discover_agent(rmw_options));
+	 RCCHECK(rmw_uros_options_set_udp_address("192.168.0.197", "8888", rmw_options));
```

Comment:
I would like to keep the agent IP and port configurable from the menuconfig

Reply:
Oops, I reverted locally because was stuck with network problems and forgot to revert. Thanks for noticing.